src/System.Security.Cryptography.OpenSsl/tests/RSAOpenSslProvider.cs

```diff
@@ -15,6 +15,11 @@ public RSA Create(int keySize)
         {
             return new RSAOpenSsl(keySize);
         }
+
+        public bool Supports384PrivateKey
+        {
+            get { return true; }
+        }
```

Comment:
I'll stick with "consistency" as my reason here, vs just saying "yeah, but I don't care much for that syntax" (maybe it'll grow on me over time)